src/chainparams.cpp

```diff
@@ -205,10 +205,10 @@ class CTestNetParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_SEGWIT].nTimeout = 1493596800; // May 1st 2017
 
         // The best chain should have at least this much work.
-        consensus.nMinimumChainWork = uint256S("0x00000000000000000000000000000000000000000000001f057509eba81aed91");
+        consensus.nMinimumChainWork = uint256S("0x00000000000000000000000000000000000000000000002830dab7f76dbb7d63");
```

Comment:
It didn't before either. nMinimum is set at the the tip, but for testnet assume valid was set 20k blocks back due to reorgs. The reason for the difference is because even in a reorg the tip will be at or equal the minimum work, but it may not contain the old tip block.